jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcRepository.java

```diff
@@ -214,4 +233,224 @@ protected List<Map<String, Object>> fetchSeekStep(SelectSeekStepN<Record> select
     protected <F extends Enum<F>> Field<?> columnToField(ColumnDescriptor<?> column, Map<F, String> fieldsMapping) {
         return column.getField() != null ? field(fieldsMapping.get(column.getField())) : null;
     }
+
+    protected <T extends Record> SelectConditionStep<T> filter(
+        SelectConditionStep<T> select,
+        List<QueryFilter> filters,
+        String dateColumn
+    ) {
+        if (filters == null) return select;
```

Comment:
Not important but we prefer to always use `{`  and `}` in statements.
```suggestion
        if (filters == null) {
            return select;
        }
```

This is more readable and allow easier refactoring

Reply:
done